app/src/main/res/layout/project_creator_info.xml

```diff
@@ -1,123 +1,112 @@
+        android:textSize="@dimen/body" />
 
       <TextView
         android:id="@+id/updates_count"
-        android:background="@drawable/text_view_rect_background"
-        android:gravity="center"
-        android:layout_centerVertical="true"
-        android:textSize="@dimen/body"
-        android:textColor="@color/ksr_soft_black"
-        android:layout_alignParentEnd="true"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        tools:text="45"/>
+        android:layout_alignParentEnd="true"
+        android:layout_centerVertical="true"
+        android:background="@drawable/text_view_rect_background"
+        android:textColor="@color/ksr_soft_black"
+        android:textSize="@dimen/body"
+        tools:text="45" />
 
     </RelativeLayout>
 
-    <include
-      layout="@layout/horizontal_line_1dp_view"
-      android:layout_height="wrap_content"
-      android:layout_width="wrap_content"
-      android:layout_marginTop="@dimen/grid_1_half"
-      android:layout_marginBottom="@dimen/grid_1_half"/>
-
     <RelativeLayout
       android:id="@+id/comments"
-      android:background="@drawable/click_indicator_light_masked"
-      android:focusable="true"
       android:layout_width="match_parent"
-      android:layout_height="@dimen/grid_7" >
+      android:layout_height="wrap_content"
+      android:focusable="true"
+      android:foreground="?android:attr/selectableItemBackground"
+      android:paddingTop="@dimen/grid_2"
+      android:paddingBottom="@dimen/grid_2">
 
       <TextView
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:layout_toStartOf="@+id/comments_count"
+        android:layout_height="wrap_content"
         android:layout_alignParentStart="true"
-        android:gravity="center_vertical"
+        android:layout_centerVertical="true"
+        android:layout_toStartOf="@+id/comments_count"
+        android:text="@string/project_menu_buttons_comments"
         android:textColor="@color/ksr_soft_black"
-        android:textSize="@dimen/body"
-        android:text="@string/project_menu_buttons_comments" />
+        android:textSize="@dimen/body" />
 
       <TextView
         android:id="@+id/comments_count"
-        android:background="@drawable/text_view_rect_background"
-        android:gravity="center"
-        android:layout_centerVertical="true"
-        android:textSize="@dimen/body"
-        android:textColor="@color/ksr_soft_black"
-        android:layout_alignParentEnd="true"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        tools:text="1,000"/>
+        android:layout_alignParentEnd="true"
+        android:layout_centerVertical="true"
+        android:background="@drawable/text_view_rect_background"
+        android:textColor="@color/ksr_soft_black"
+        android:textSize="@dimen/body"
+        tools:text="1,000" />
```

Comment:
Is this just a temporary value to preview the design?

Reply:
yep! any properties with the `tools` namespace are only visible when previewing the layout

Reply:
Awesome! 👍